themes/default/content/blog/customer-managed-deployment-agents-launch/index.md

```diff
@@ -0,0 +1,138 @@
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-01-08T08:06:12-07:00
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: Customer-Managed agents for Pulumi Deployments is now available, allowing users to self-host deployment agents and get the same power and flexibility of Pulumi Deployments. 
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - arun-loganathan
+    - german-lena 
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+
+---
+
+At Pulumi, we constantly strive to build solutions that not only simplify infrastructure management but also align with the diverse and specific needs of our customers. To that end, we're thrilled to announce the general availability of Customer-Managed Agents for Pulumi Deployments. 
+
+<!--more-->
+
+## Primer on Pulumi Deployments
+
+[Pulumi Deployments](/docs/pulumi-cloud/deployments/) is a fully managed platform for managing infrastructure at scale, providing easy-to-use building blocks that can be used as a simple CI/CD platform or to build a sophisticated self-serve platform to support thousands of engineers. [Traditional CI/CD platforms](/docs/pulumi-cloud/deployments/versus) focus on the git push workflows which fall short when managing infrastructure where the same code base is used to create new infrastructure with no explicit git push event.  Pulumi Deployments go beyond the git push based workflows to offer deployment triggers based on REST API, click to deploy in the Pulumi Cloud UI, review Stacks, remote Automation API, etc. 
+
+Since its launch, Pulumi Deployments have transformed how organizations manage their infrastructure. offloading the execution of Pulumi Infrastructure as Code (IaC) actions - such as previews, updates, destroys, and refreshes - to Pulumi-managed compute. Our customers have used it to deploy and manage mission-critical services, driving efficiencies and reducing the operational burden of deploying and managing infrastructure. 
```

Comment:
```suggestion
Since its launch, Pulumi Deployments has transformed how organizations manage their infrastructure, offloading the execution of Pulumi Infrastructure as Code (IaC) actions - such as previews, updates, destroys, and refreshes - to compute Pulumi manages. Our customers have used it to deploy and manage mission-critical services, driving efficiencies and reducing the operational burden of deploying and managing infrastructure. 
```